### PR TITLE
Feat input

### PR DESCRIPTION
**Pull request by Samarth-5109**

### CODE REVIEW

**Comment by Samarth-5109** [comment]
added input form and form with validation

**Comment by Samarth-5109** [comment]
Screenshot (48)

File: components/inputs/inputs.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    font-weight: bold;
+}
+
+.btn{
+    background-color: var(--primary-color);
+    color: var(--secondary-color);
+    padding: 12px;
+    border-radius: 5px;
+    /* margin:1rem; */
+}
```
Comment (prath-dev) [inline components/inputs/inputs.css]: In btn padding try to use rem if it is more than 10px ... as vikram sir also suugested this in his css session..  An also try to delete commented code

**Review by MonikaJhingan — COMMENTED** [review]
Code.looks.good.to be merged but add meta author and description tags in later projects.

**Review by MonikaJhingan — APPROVED** [review]

File: components/inputs/inputs.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;1,100&family=Shippori+Antique&display=swap");
+@import url("../../styles/doc.css");
+
+.btn-wrapper {
+  margin-top: 3rem;
+}
+
+.input-form {
```
Comment (SuyashTiwariCoder) [inline components/inputs/inputs.css:8]: You can remove unused classes

**Review by prath-dev — APPROVED** [review]

File: components/inputs/inputs.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  /* background-color: aqua; */
+}
+
+.required-field label::after {
+  content: "*";
+  color: red;
+}
```
Comment (SuyashTiwariCoder) [inline components/inputs/inputs.css:15]: use variables for colors

File: components/inputs/inputs.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  padding: 10px;
+}
+
+.required {
+  color: var(--primary-color);
+  font-weight: bold;
+}
+
+.success {
+  border: 2px solid green;
```
Comment (SuyashTiwariCoder) [inline components/inputs/inputs.css:31]: again .. use variables for colors

File: components/inputs/inputs.html [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+            <div class="sidebar">
+                <p class="sidebar-title">component list</p>
+            <ul class="sidebar-items">
+                <li class="list-item"><a href="alerts.html">Alerts</a></li>
+                <li class="list-item"><a href="avatar.html">Avatars</a></li>
+                <li class="list-item"><a href="badges.html">Badges</a></li>
+                <li class="list-item"><a href="button.html">Buttons</a></li>
+                <li class="list-item"><a href="cards.html">Cards</a></li>
+                <li class="list-item"><a href="images.html">Image</a></li>
+                <li class="list-item"><a href="">Inputs</a></li>
```
Comment (SuyashTiwariCoder) [inline components/inputs/inputs.html:36]: use # in the links that are not used right now . Make it a dead link

File: components/inputs/inputs.html [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+            </div>
+            <div class="types-container">
+                <form action="input" class="input-form">
+                    <div class="input-box required-field">
+                        <label class="input-label" class="email-label " for="email">Email address</label></br>
+                        <input type="email" class="email-input" name="email" placeholder="enter your email" required>
+                    </div>
+                    <div class="input-box ">
+                        <label class="input-label"for="textarea">Text Area</label></form></br>
+                        <textarea class="text-area"name="textarea" id="" cols="50" rows="7"></textarea>
```
Comment (SuyashTiwariCoder) [inline components/inputs/inputs.html:74]: You can remove attibutes that are not in used .

**Review by SuyashTiwariCoder — CHANGES_REQUESTED** [review]

**Pull request merged**